src/compiler/preprocess/types.ts

```diff
@@ -22,6 +22,7 @@ export type MarkupPreprocessor = (options: {
 export type Preprocessor = (options: {
 	content: string;
 	attributes: Record<string, string | boolean>;
+	source: string;
```

Comment:
Would it make sense to call it `markup`? I can imagine folks confused with the difference between `content` and `source`

Reply:
Of course, I don't see any problem with this.
Do you want me to change this now or do you want to wait the feedback of others contributors?

Reply:
I would wait for the feedback of others! I was just coming here to say that I started to like `source`, but let's see what others have to say

Reply:
Ooops, I did the exact inverse, I started to like `markup` so I changed it.
I will let the `markup` while waiting for feedbacks from others :-)

Reply:
I reverted to `source` because `markup` conflicts with `markup` preprocessors and when I try to use this name in `svelte-preprocess` it emits eslint `no-shadow` errors.

Reply:
I'd rather have this `markup` again to be honest, `content` and `source` are both too generic/similar. If the problem is only the ESLint error within `svelte-preprocess`, then you can easily work around that by doing a rename while destructuring the input, like `{content, attributes, markup: source}` .

Reply:
OK for `markup`. I will workaround in `svelte-preprocess`.
I will update my branch.